Makefile

```
@@ -6,6 +6,8 @@ PYTEST_COV_ARGS=--cov=$(PROJECT) --cov-config=../pyproject.toml \
 			--pyargs ${PYTEST_EXTRA}
 BLACK_FILES=$(PROJECT) setup.py doc/conf.py examples
 BLACKDOC_OPTIONS=--line-length 79
+DOCFORMATTER_FILES=$(PROJECT) setup.py doc/conf.py examples
+DOCFORMATTER_OPTIONS=--recursive --pre-summary-newline --wrap-summaries 79 --wrap-descriptions 79
```

```suggestion
DOCFORMATTER_OPTIONS=--recursive --pre-summary-newline --make-summary-multi-line --wrap-summaries 79 --wrap-descriptions 79
```
Do we want to force `--make-summary-multi-line`? This will turn every description (even short ones) into:
```python
"""
This is a description.
"""
```

---

Yes, it looks cleaner.

---

Just find that [PEP 257](https://www.python.org/dev/peps/pep-0257/) says:

> Unless the entire docstring fits on a line, place the closing quotes on a line by themselves.

---

Hmm, seems like `docformatter` didn't implement that [one-line docstring PEP 257 guideline](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) correctly? I think it's ok to just force multi-line though, as long as there's an automated solution.

---

/format